crates/astria-sequencer-client/src/client.rs

```diff
@@ -193,21 +122,20 @@ impl Client {
     /// # Errors
     ///
     /// - If calling the tendermint RPC endpoint fails.
-    pub async fn submit_transaction_commit(&self, tx: Signed) -> eyre::Result<tx_commit::Response> {
+    async fn submit_transaction_commit(&self, tx: Signed) -> eyre::Result<tx_commit::Response> {
```

Comment:
I think this could just return the error of `broadcast_tx_commit` instead of wrapping it in `eyre`